nbclassic/static/style/ipython.less

```
@@ -9,4 +9,4 @@
 @import "../base/less/style.less";
 
 // notebook
-@import "../notebook/less/style_noapp.less";
+@import "../nbclassic/less/style_noapp.less";
```

Updating the path to this file resolved the 404 error previously occurring since the folder name is still notebook. I'm not sure if it's best to update the folder name or if it's okay to keep it as 'notebook'? 
```suggestion
@import "../notebook/less/style_noapp.less";
```